LayoutTests/fast/images/spatial-image-controls-expected.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<style>
+    img {
+        width: 200px;
+        height: 200px;
+    }
+    div {
+        width: 200px;
+        position: relative;
+    }
+    #controls {
```

Comment:
This effectively copies UA stylesheet's rule... that's not great but I don't think there is any other way.